### DISCUSSION TITLE
我宣布

### DISCUSSION THREAD

**Discussion by Qweulin** [share121/inter-knot#2400] [其他]
千年科技学院入侵绳网![lol](https://i.ibb.co/NyT8ZZm/20240812220702.jpg)

**Comment by Qweulin** [comment]
🤓🤓🤓![and](https://i.ibb.co/yfPm32k/Image-1723651753393.png)

**Reply by Qweulin** [reply]
![hello](https://i.ibb.co/CKr3Sd7/Image-1718689934585.jpg)

**Comment by Qweulin** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Comment by YuTinMIn** [comment]
让sensei我看看怎么回事（

**Comment by xinyuanyoo** [comment]
这里也有sensei

**Comment by Phaethon-0019** [comment]
千年科技难道不该放优香吗

**Reply by Qweulin** [reply]
sensei我喜欢日步美(⁠｡⁠•̀⁠ᴗ⁠-⁠)⁠✧

**Comment by blqdswkl** [comment]
好有感觉

**Comment by bushigemengsishuihaiyinga** [comment]
可是千年的话放三一的学生是什么鬼？新活？

**Comment by BJ-cmyk** [comment]
丸辣

**Comment by ff688** [comment]
浮士德干的😁

**Comment by Herrscher-of-Origin** [comment]
是千年的爱丽丝同学吗

**Comment by Minapocalis** [comment]
完了
![1725712249245](https://github.com/user-attachments/assets/3c7a73a8-d6e5-4bf9-a610-5cad43e9e7f0)

**Comment by BJ-cmyk** [comment]
![1726063472430](https://github.com/user-attachments/assets/da8778f0-5533-4ced-ac84-b7ba0b86f28e)
爱丽丝可爱捏
